### DISCUSSION TITLE
A native exception occurred during launch

### DISCUSSION THREAD

**Discussion by dreamfly999** [microsoft/vscode-discussions#823] [Extension Development QnA]
when I install the recent version, such as 1.82.2，I will get error when I connected to remote server by vscode. Just when I use the 1.52.1 is normal, but some plug-ins are not available. I don't know is it right ask for help here, but I really want to solve the problem.
image
error message:
A native exception occurred during launch (EBUSY: resource busy or locked, open '.\pipe\conpty-5995336.540991683-in')

**Comment by digitarald** [comment]
Please file an issue from within VS Code (Report Issue) or via https://github.com/microsoft/vscode
